content/applications/productivity/voip.rst

```diff
@@ -1,14 +1,77 @@
 :show-content:
-:hide-page-toc:
-:show-toc:
+
+.. |VOIP| replace:: :abbr:`VoIP (Voice over Internet Protocol)`
 
 ===================================
 VoIP (Voice over Internet Protocol)
 ===================================
 
+|VOIP| in Odoo enables businesses to handle calls over the internet by integrating directly with
+Odoo apps like **CRM**, **Helpdesk**, and more. Calls and messages are logged within these apps,
+keeping records linked to customer interactions.
+
+Overview
+========
+
+Users can make and receive calls, track communication history, and automate call routing based on
+predefined rules. Features like call recording and analytics provide insights into call volume and
+response times, helping teams track communication efficiency.
+
+.. cards::
+   .. card:: VoIP widget
+      :target: voip/voip_widget
+
+      Get oriented with the features of the VoIP widget, like what actions can be taken during a
+      call.
+
+   .. card:: Devices and integrations
+      :target: voip/devices_integrations
+
+      Learn about accessing the VoIP widget from different devices (like phones) and apps (like
+      Linphone).
+
+   .. card:: Make, receive, transfer, and forward calls
+      :target: voip/transfer_forward
+
+      Learn how to use the VoIP widget to take actions, like call forwarding.
```

Comment:
Note for next PR:
I'm not sure what the purpose is having two docs on using the VoIP widget, seems like all of the content from `transfer_forward.rst` can be moved to `voip_widget.rst`
```suggestion
```